stylize/clang_formatter.py

```diff
@@ -34,3 +35,14 @@ def run(self, args, filepath, check=False):
             proc.communicate()
             md5_after = file_md5(filepath)
             return md5_before != md5_after
+
+    def get_command(self):
+        if os.system("command -v clang-format >/dev/null") == 0:
+            return "clang-format"
+        # Run the next command in bash, as we need the bash builtin, compgen
+        possible_command = subprocess.Popen(["bash", "-c", "compgen -A function -abck | grep -E clang-format-[0-9]+\.[0-9]+ | tail -n 1"],
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE).stdout.read().decode("utf-8").strip()
```

Comment:
This was the only way I could find to capture shell output in python... Its a little confusing :cry: 
I'm not sure if there's a better way, os.system seems to return just the exit code.